### PR TITLE
Offworlder Captains

### PR DESCRIPTION
**Pull request by MortalRed**
Based on the responses in https://forums.aurorastation.org/topic/18026-offworlder-captains/ , offworlder humans being restricted from captaincy was a temporary measure from when they were first introduced and plenty of time has passed since then. This PR removes that restriction.

### CODE REVIEW

**Comment by MortalRed** [comment]
!review

**Comment by SleepyGemmy** [comment]
lore approved?

File: html/changelogs/MortalRed-offieCaptains.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Offworlder Humans can now be Captains"
```
Comment (SleepyGemmy) [inline html/changelogs/MortalRed-offieCaptains.yml:41]:
```suggestion
  - rscadd: "Offworlder humans can now be a captain."
```

**Review by alsoandanswer — APPROVED** [review]

**Comment by Caelphon** [comment]
This is okay, however, I specifically wish for Arrow to also go through this and give a thumbs up.

**Review by SleepyGemmy — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**